packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Snippets/JsonElementSnippets.cs

```diff
@@ -90,5 +90,14 @@ public static ScopedApi<bool> TryGetInt64(this ScopedApi<JsonElement> jsonElemen
             var invocation = new InvokeMethodExpression(jsonElement, nameof(JsonElement.TryGetInt64), [new DeclarationExpression(longValueDeclaration, true)]);
             return invocation.As<bool>();
         }
+
+        internal static ScopedApi<bool> TryGetValue<T>(this ScopedApi<JsonElement> jsonElement, string methodName, out ScopedApi<T> value)
```

Comment:
I think it would be better to mirror the System.Text.Json APIs as much as possible so that the snippet is just a wrapper around invoking a specific method. This would follow the pattern we have for TryGetInt32, TryGetInt64, etc.

Reply:
sure, done.